tests/client/test_query.py

```diff
@@ -1,5 +1,6 @@
 # SPDX-License-Identifier: GPL-3.0+
 
+from collections.abc import Mapping, Collection
```

Comment:
Nitpick: new line between stdlib imports and module imports.

Reply:
Added.